app/views/partials/homepage.html

```diff
@@ -83,4 +85,33 @@ <h2 class="text-center">Les aides simulées</h2>
       </div>
     </div>
   </div>
+  <div class="frame-aides-homepage">
+    <h2 class="text-center">Prestations locales</h2>
+    <div ng-repeat="partenaireLocal in partenairesLocaux">
+      <hr>
+      <div class="row">
+        <div class="text-center col-md-2">
+          <img ng-src="/img/{{ partenaireLocal.imgSrc }}" alt=""/>
+          <h2>{{ partenaireLocal.label }}</h2>
+        </div>
+        <div class="col-md-10">
+          <div ng-repeat="droit in partenaireLocal.prestations">
+            <div class="clearfix" ng-if="showDetails && $index % 2 == 0"></div>
```

Comment:
Ok, I doubled check with Safari. It is working well, except for this (that I already noticed with a friend's phone):

![image](https://cloud.githubusercontent.com/assets/11834997/13783975/cc71366e-eacd-11e5-9199-40c1a00774ec.png)